core/src/main/java/com/airbnb/aerosolve/core/transforms/ListTransform.java

```diff
@@ -34,4 +35,13 @@ public void doTransform(FeatureVector featureVector) {
       transform.doTransform(featureVector);
     }
   }
+
+  @Override
+  public void doTransform(Stream<FeatureVector> featureVectors) {
```

Comment:
why `Stream` instead of something like `List`, I feel it can be tricky for debugging when using `Stream` as type of input parameter due to its lazy evaluation. perhaps you can convert this to stream inside?